include/boost/geometry/strategies/intersection_result.hpp

```diff
@@ -26,7 +31,7 @@ namespace boost { namespace geometry
     \ingroup overlay
     \see http://gis.hsr.ch/wiki/images/3/3d/9dem_springer.pdf
 */
-struct de9im
+/*struct de9im
```

Comment:
Yes, I think so, already 7 or 8 years ago.
I'm very fine with removing it now.

The new code looks very good.